lib/experimental/interactivity-api/directive-processing.php

```diff
@@ -19,120 +19,43 @@
  * @return array The parsed block.
  */
 function gutenberg_interactivity_process_directives( $parsed_block, $source_block, $parent_block ) {
```

Comment:
We can rename this filter again because it's not processing the directives anymore, only marking the root blocks.

Also, the DocBlock description needs to be updated.